app/components/nav-bar.js

```diff
@@ -1,4 +1,5 @@
 import Ember from 'ember';
+import $ from 'jquery';
```

Comment:
Don't import jquery directly. Use `this.$('<selector>')` inside a component to access a component-level jquery object.